apps/charterafrica/src/payload/collections/Tools.js

```diff
@@ -0,0 +1,307 @@
+      admin: {
+        readOnly: true,
+      },
+      label: { en: "Avatar URL", fr: "URL d'avatar", pt: "URL de avatar" },
+    },
+    {
+      name: "lastCommit",
+      type: "group",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Last Commit",
+        fr: "Dernier commit",
+        pt: "Último commit",
+      },
+      fields: [
+        {
+          name: "message",
+          type: "text",
+          admin: {
+            readOnly: true,
+          },
+          label: { en: "Message", fr: "Message", pt: "Mensagem" },
+        },
+        {
+          name: "author",
+          type: "text",
+          admin: {
+            readOnly: true,
+          },
+          label: { en: "author", fr: "auteur", pt: "autor" },
+        },
+        {
+          name: "committedDate",
+          type: "date",
+          admin: {
+            readOnly: true,
+            date: {
+              pickerAppearance: "dayAndTime",
+            },
+          },
+          label: {
+            en: "Comitted Date",
+            fr: "Date d'arrivée",
+            pt: "Data emodificada",
+          },
+        },
+      ],
+    },
+    {
+      name: "stars",
+      type: "number",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Stars",
+        fr: "Étoiles",
+        pt: "Estrelas",
+      },
+    },
+    {
+      name: "views",
+      type: "number",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Views",
+        fr: "Vues",
+        pt: "Visualizações",
+      },
+    },
+    {
+      name: "forks",
+      type: "number",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Forks",
+        fr: "Fourches",
+        pt: "Forks",
+      },
+    },
+    {
+      name: "organisation",
+      type: "relationship",
+      admin: {
+        readOnly: true,
+      },
+      relationTo: ORGANIZATION_COLLECTION,
+      label: {
+        en: "Organisation",
+        fr: "Organisation",
+        pt: "Organização",
+      },
+    },
+    {
+      name: "contributors",
+      type: "relationship",
+      hasMany: true,
+      admin: {
+        readOnly: true,
+      },
+      relationTo: CONTRIBUTORS_COLLECTION,
+      label: { en: "Contributors", fr: "Contributeurs", pt: "Colaboradores" },
+    },
+    {
+      name: "externalUpdatedAt",
+      type: "date",
+      admin: {
+        readOnly: true,
+        date: {
+          pickerAppearance: "dayAndTime",
+        },
+      },
+    },
+    donors(),
+    {
+      name: "partners",
+      type: "array",
+      admin: {
+        readOnly: true,
+        initCollapsed: true,
+      },
+      label: { en: "Partners", fr: "Les partenaires", pt: "Parceiros" },
+      fields: [
+        {
+          name: "name",
+          type: "text",
+          label: { en: "Name", fr: "Nom", pt: "Nome" },
+          required: true,
+        },
+      ],
+    },
+    {
+      name: "originLocation",
```

Comment:
What is the difference between this and `deploymentLocation`?

Reply:
Location of origin is a location where it is initially based but deployment location is where it is used .

For instance Pesa Yetu's original location can be South Africa but deployed for use in Tanzania

Reply:
This doesn't make any sense @koechkevin 
> ... of origin is a location where it is initially based ...

it what? The team developing it, the source code repository, the registered office of the organisation developing it, etc.

PesaYetu was developed to visualise Kenyan government budget so location of first deployment would be Kenya. I'm still not sure what PesaYetu location would be or if it make sense.

Reply:
@kilemensi 

Here's an explanation of the same by Sarah https://code4africa.slack.com/archives/C03SB274J4B/p1685108907238259